benches/languages.rs

```diff
@@ -10,10 +10,132 @@ criterion_group!(
     // encapsulates CI noise we saw in
     // https://github.com/timberio/vector/issues/5394
     config = Criterion::default().noise_threshold(0.05);
-    targets = benchmark_parse_syslog
+    targets = benchmark_add_fields, benchmark_parse_json, benchmark_parse_syslog
 );
 criterion_main!(benches);
 
+// Add two fields to the event: four=4 and five=5
+fn benchmark_add_fields(c: &mut Criterion) {
+    let configs: Vec<(&str, &str)> = vec![
+        (
+            "remap",
+            r#"
```

Comment:
I'm a little late to this one, but [indoc](https://github.com/dtolnay/indoc) could help readability a lot here.

Reply:
TIL! This will certainly come in handy in the future.